operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -76,6 +76,14 @@ spec:
 #    ---
 #    strategy: ""
 
+##########
+# Determines whether to show links of Grafana and Traces or not depending on user
+# access to Istio control plane pods. Kiali won't show Grafana and Tracing integration
+# in case the user doesn't have permissions to list pods of IstioNamespace.
+# When empty, its default will be `true`.
+#    ---
```

Comment:
if this setting is a top level setting, then format this "---" line - all of these separators are supposed to be on the same indentation as the level that the settings are at. Helps keep a visual clue as to what level these settings are at.
```suggestion
#  ---
```